### ISSUE TITLE
2024-04-22

### ISSUE THREAD

**Issue by JavaStudentAlex** [MAGNET4Cardiac7T/magnet-pinn#5]
# Progress
- #2 
# Agenda
- Do field files have the same format?(all .txt or all .h5)
- Just one instance of class for processing multiple simulations?
# Future work
- no need to recheck the mask of dipoles
- start from the grid structure
- voxelize dipoles only once
- give different resolution options
- start documentation
